beacon-chain/state/state-native/getters_misc.go

```diff
@@ -67,15 +68,19 @@ func (b *BeaconState) forkVal() *ethpb.Fork {
 }
 
 // HistoricalRoots based on epochs stored in the beacon state.
-func (b *BeaconState) HistoricalRoots() [][]byte {
+func (b *BeaconState) HistoricalRoots() ([][]byte, error) {
+	if b.version > version.Bellatrix {
```

Comment:
This is not accurate, the historical roots field still exists in the state. Just that it is now 'frozen'